zeppelin-interpreter/src/main/java/org/apache/zeppelin/display/Input.java

```diff
@@ -373,7 +373,11 @@ public static String getSimpleQuery(Map<String, Object> params, String script, b
         }
       } else {
         // single-selection
-        expanded = value.toString();
+        if (value == null) {
```

Comment:
[StringUtils.defaultString](http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#defaultString%28java.lang.String%29) looks more elegant.

Reply:
Thanks for the review @Reamer  I have updated it